.github/workflows/build-test-deploy.yml

```diff
@@ -240,32 +240,6 @@ jobs:
       - run: |
           cachix watch-exec --jobs 16 --compression-level $CACHIX_COMPRESSION_LEVEL composable-community nix -- build .#spell-check --no-update-lock-file --show-trace -L
 
-  mdbook-check:
```

Comment:
may you link here task (gh or cu) about making check for ds docs. if that will not be made - it will rot very soon. thank you

Reply:
Yes it was already on my todolist for after the initial version, but I've made a ticket for it here and assigned it to myself: https://app.clickup.com/t/321vvt2